birdhouse/scripts/bootstrap-instance-for-testsuite

```diff
@@ -18,15 +18,17 @@ set -x
 # Need write-access to DATASET_ROOT (/data/datasets/).
 $THIS_DIR/bootstrap-testdata
 
-# Index Thredds catalog.
-# Need to open temporary Thredds "testdata/secure/" on local PAVICS host to anonymous group.
-# Only crawl the subset enough to pass canarie-api monitoring
-# see config/canarie-api/docker_configuration.py.template
-$THIS_DIR/trigger-pavicscrawler target_files=birdhouse/testdata/flyingpigeon/cmip5
+if [ -n "$(docker ps --filter name=solr)" ]; then
+  # Index Thredds catalog.
+  # Need to open temporary Thredds "testdata/secure/" on local PAVICS host to anonymous group.
```

Comment:
I think some tests were using `testdata/secure/` to validate some protected THREDDS file access... 
To see if anything arises from the CI.